docs/api/browser-window.md

```diff
@@ -540,15 +540,15 @@ win.loadURL('https://github.com')
 
 #### `win.webContents`
 
-The `WebContents` object this window owns. All web page related events and
+A `WebContents` object this window owns. All web page related events and
```

Comment:
@enlight To maintain scope of this PR things like readonly should be left to a new set of documentation / parser PR's